README.md

```diff
@@ -5,6 +5,8 @@ Description: >-
   built on. These docs teach you how to contribute!
 ---
 
+*test change to showcase PR inline comments - do not merge*
+
```

Comment:
Your comments can go anywhere in the file that is being changed as part of the PR.